resources/scripts/pytest_otel/README.md

```diff
@@ -0,0 +1,65 @@
+
+Features
+--------
+
+pytest-otel plugin for report APM traces about test executed.
+
+
+Requirements
+------------
+
+* opentelemetry-api == 1.2
+* opentelemetry-exporter-otlp == 1.2.0
+* opentelemetry-sdk == 1.2.0
+
+
+Installation
+------------
+
+You can install "pytest-otel" via `pip` or using the `setup.py` script.
+
+```
+git checkout https://github.com/elastic/apm-pipeline-library
+cd apm-pipeline-library/resources/scripts
+pip install ./pytest_otel
+```
+
+Usage
+-----
+
+pytest_otel is configured by adding some parameters to the pytest command line here are the descriptions
+
+* --endpoint: URL for the OpenTelemetry server. (Required). Env variable `OTEL_EXPORTER_OTLP_ENDPOINT`
+* --headers: Additional headers to send (i.e.: key1=value1,key2=value2). Env variable `OTEL_EXPORTER_OTLP_HEADERS`
+* --service-name: Name of the service. Env variable `OTEL_SERVICE_NAME`
+* --session-name: Name for the Main span reported.
+* --trace_id: Trace parent id. Env variable `TRACE_ID` see https://www.w3.org/TR/trace-context-1/#trace-context-http-headers-format
+* --insecure: Disables TLS. Env variable `OTEL_EXPORTER_OTLP_INSECURE`
+
+```bash
+cd pytest_otel
+pytest --endpoint https://apm.example.com:8200 \
+       --headers "Authorization=Bearer ASWDCcCRFfr" \
+       --service-name pytest_otel \
+       --session-name='My_Test_cases' \
+       --trace_id=00-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331-01 \
```

Comment:
The pattern of this value looks like a `traceparent` more than a `trace_id`

Reply:
it is a traceparent, I did not found documentation or samples about the content of `TRACE_ID` environment variable so I guessed that it is the content of the header `traceparent` as it is specify by https://www.w3.org/TR/trace-context-1/#trace-context-http-headers-format

Reply:
```suggestion
       --traceparent=00-0af7651916cd43dd8448eb211c80319c-b7ad6b7169203331-01 \
```